packages/embed/e2e/spec/functional/close-on-keyboard.spec.ts

```diff
@@ -0,0 +1,31 @@
+describe('Close on Keyboard Esc Event', () => {
+  it('Should close the form with keyboard event inside the Iframe', () => {
+    cy.visit('/popup-html.html')
+
+    cy.get('#button').click()
+
+    cy.get('iframe').should('be.visible')
+
+    cy.get('iframe').then(($iframe) => {
+      const $body = $iframe.contents().find('body')
```

Comment:
I'm not familiar with this syntax, do we prefix $ on other cypress test variables?

Reply:
I think is a cypress pattern, I see we used it in other tests too